docs/wiki/elpis-dev-recipe.md

```diff
@@ -2,41 +2,66 @@
 
 This guide can assist in setting up directory structures to load repositories into a Docker container, enabling you to develop code and interact with the changes. This guide doesn't cover testing.
 
+The recommended folder structure is to have a `~/sandbox` folder inside your user directory. This can contain the `elpis` Git repository, the `espnet` repository and the `state` folder to view the state that the program generates.
 
 ## Prepare your local dirs
 
 Get the repos for developing, `git pull` etc if you need.
 
-```
-mkdir ~/elpis-sandbox
-cd ~/elpis-sandbox
+```shell
+mkdir ~/sandbox
+cd ~/sandbox
 git clone --depth=1 https://github.com/CoEDL/elpis.git
-git clone --depth=1 https://github.com/CoEDL/elpis-gui.git
 git clone --depth=1 -b elpis https://github.com/persephone-tools/espnet
 ```
 
-
 ## Build the GUI
 
-The Docker container has a build of the React app GUI in it, but if you are cloning the GUI repository and replacing the directory in the container with the local repository, the app build directory won't exist in the container (it is excluded from version control so it isn't in what you cloned). Run these commands to install the NPM libraries required, and to build a production version of the GUI into the `elpis-gui/build` dir. If you are developing the GUI, replace `npm run build` with `npm run watch` and then when you make changes to the GUI code, you'll get an automatic rebuild of the app. You will need to manually reload the browser that has the interface.
+The Docker container has a build of the React app GUI in it, but if you are cloning the GUI repository and replacing the directory in the container with the local repository, the app build directory won't exist in the container (it is excluded from version control so it isn't in what you cloned). Run these commands to install the NPM libraries required, and to build a production version of the GUI into the `gui/build` dir. If you are developing the GUI, replace `npm run build` with `npm run watch` and then when you make changes to the GUI code, you'll get an automatic rebuild of the app. You will need to manually reload the browser that has the interface.
```

Comment:
This can be updated to account for the mono-repo. Something like this perhaps?

> The Docker container has a build of the React app GUI in it. If you are cloning the `elpis` repository and working on the GUI, run these commands to enable changes to the GUI code to be reloaded in the browser.